conduct.md

```diff
@@ -14,7 +14,32 @@
 * Participation
 * Decisions
 * Postmortems
-* Conflicts
+
+## Conflicts
+
+It is inevitable that conflicts will arise when working with a large group of
+people. It is essential, however, to resolve these conflicts in a professional
+manner while keeping in mind the concerns of the people involved.
+
+* **Acceptable Conduct (A):**
+
```

Comment:
Just one tiny issue: remove the blank line between **Acceptable Conduct (A):**, * *Members shall* and the body since this is the format in the style guide. It is better to just stay consistent even though it does appear the same way when viewing it

Reply:
sounds good! I fixed all formatting issues to be like the template.